docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,170 @@
+
+```js
+Excel.createWorkbook();
+```
+
+`createWorkbook` takes in an optional string. This string is an .xlsx file in base64 encoding. Assuming the string argument is a valid .xlsx file, the resulting workbook will be a copy of that file. You can get your add-in’s current workbook as a base64-encoded string by using [file slicing](/javascript/api/office/office.document#getfileasync-filetype--options--callback-). The [FileReader](https://developer.mozilla.org/docs/Web/API/FileReader) class can be used to convert a file into the required base64-encoded string, as demonstrated in the following example. 
+
+```js
+var myFile = document.getElementById("file");
+var reader = new FileReader();
+
+reader.onload = (function (event) {
+	Excel.run(function (context) {
+		// strip off the metadata before the base64-encoded string
+		var startIndex = event.target.result.indexOf("base64,");
+		var mybase64 = event.target.result.substr(startIndex + 7);
+
+		Excel.createWorkbook(mybase64);
+		return context.sync();
+	}).catch(errorHandlerFunction);
+});
+
+// read in the file as a data URL so we can parse the base64-encoded string
+reader.readAsDataURL(myFile.files[0]);
+```
+
+## Protection
+
+Your add-in can control a user's ability to edit the worksheet structure. The workbook's `protection` property is a [WorkbookProtection](/javascript/api/excel/excel.workbookprotection) object with a `protect()` method. The following example shows a basic scenario toggling the protection of the workbook's structure. 
+
+```js
+Excel.run(function (context) {
+	var workbook = context.workbook;
+	workbook.load("protection/protected");
+
+	return context.sync().then(function() {
+		if (!workbook.protection.protected) {
+			workbook.protection.protect();
+		}
+	});
+}).catch(errorHandlerFunction);
+```
+
+The `protect` method also has an optional string parameter, `password`. This string represents the password needed for a user to bypass protection and change the workbook's structure.
+
+Protection can also be set at the worksheet level. Protection on the worksheet guards against unwanted data editing. For more information, the Data protection section of the [Work with Worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md#data-protection) article.
+
+The article [Protect a workbook](https://support.office.com/article/Protect-a-workbook-7E365A4D-3E89-4616-84CA-1931257C1517) has more information about workbook protection and how to change it through the Excel UI.
+
+## Document properties
+
+A workbook has access to the Office file metadata, which is known as the [document properties](https://support.office.com/article/View-or-change-the-properties-for-an-Office-file-21D604C2-481E-4379-8E54-1DD4622C6B75). The workbook's `properties` property is a [DocumentProperties](/javascript/api/excel/excel.documentproperties) object containing these metadata values. The following example shows how to set the author property.
```

Comment:
- Consider rephrasing first sentence to -> "A Workbook object provides access to..."
- Start second sentence with "The Workbook object's..."  (anytime we're referring to properties or methods, it should be "the Workbook object's properties | methods")
- format **author** as bold -> **author** property